README.md

```diff
@@ -530,17 +530,19 @@ with the `mod_auth_gssapi` module.
 
 ## Nexus
 
-### Nexus For npm
+### Nexus For Java Script
 
-The npm package manager functionality relies on [Nexus Repository Manager 3][nexus-docs] to store
-npm dependencies. The Nexus instance will have an npm group repository (e.g. `cachito-js`) which
-points to an npm hosted repository (e.g. `cachito-js-hosted`) and an npm proxy repository
-(e.g. `cachito-js-proxy`) that points to registry.npmjs.org. The hosted repository will contain all
-non-registry dependencies and the proxy repository will contain all dependencies from the npm
-registry. The union of these two repositories gives the set of all the npm dependencies ever
+The Java Script(JS) package managers (npm, yarn) functionality relies on
+[Nexus Repository Manager 3][nexus-docs] to store JS dependencies. The Nexus instance will have a
+JS group repository (e.g. `cachito-js`) which points to a JS hosted repository (e.g.
+`cachito-js-hosted`) and a JS proxy repository
+(e.g. `cachito-js-proxy`) that points to registry.npmjs.org or registry.yarnpkg.com according to
```

Comment:
`cachito-js-proxy` points to registry.npmjs.org for both npm and yarn. The reason why this works is that the 2 hostnames actually point to the same server (i.e. registry.yarnpkg.com is a CNAME for registry.npmjs.org).

Reply:
This. And it would be nice to mention it here, if w eare going to describe this process in such detailed level.